cmd/cephfs/main.go

```diff
@@ -43,6 +43,13 @@ func init() {
 
 func main() {
 
+	err := util.ValidateDriverName(*driverName)
+	if err != nil {
+		klog.Fatalln(err)
+	}
+	//update plugin name
+	cephfs.PluginFolder = cephfs.PluginFolder + *driverName
+
 	cp, err := util.CreatePersistanceStorage(cephfs.PluginFolder, *metadataStorage, *driverName)
```

Comment:
There is a breaking change when using metaDataStoage as the "node" and the drivername is changed from the default. The node level storage will now be queried under the new name, and hence if it was present under the older name it would not be found and older volumeID maps would not be found.

Reasoning is due to changing driverName input to NewCachePersister, when metaDataStore is "node"

As this is just a node level storage, we should retain the older name as the drivername that is passed to this function. It would still break in case some usage passed in the option "--drivername" to be different than the default, which needs separate discussion/consolidation anyway (as in validation itself may break and hence driver would never come up, and we possibly need to note any migration steps hence).

(Applies to rbd as well)

Reply:
this also won't do any breaking change as the driver name validation is not failing,
current code  changes should work with old  cluster,  i will validate these changes  and update  here